README.md

```diff
@@ -1 +1,74 @@
-# shiny.i18n
\ No newline at end of file
+Repo documentation
+================
+
+<link href="http://fonts.googleapis.com/css?family=Lato:300,700,300italic|Inconsolata" rel="stylesheet" type="text/css"> <link href='docs/style.css' rel='stylesheet' type='text/css'>
+
+shiny.i18n
+==========
+
+Shiny applications internationalization made easy
+
+Change languages and formats easy with shiny.i18n.
+
+<!-- #Basic tutorial article is available on [Appsilon Data Science blog](your_future_art_link). -->
+<!-- Live demo link below 
+<p style="text-align: center; font-size: x-large;">
+<a href="http://appsilondatascience.com/demos">Live demo</a>
+</p>
+-->
+
+Source code
+-----------
+
+This library source code can be found on [Appsilon Data Science's](http://appsilondatascience.com) Github: <br> <https://github.com/Appsilon/shiny.i18n/>
+
+<script>
```

Comment:
Are you sure that those scripts should be here? for example in `sc` README I cannot find it: https://raw.githubusercontent.com/Appsilon/shiny.collections/master/README.md

Reply:
Because this are in Rmd files: https://raw.githubusercontent.com/Appsilon/shiny.collections/master/README.Rmd

Reply:
Oh, I know what you mean! This is fixed now with: https://github.com/Appsilon/shiny.i18n/pull/5